src/main/java/fr/techad/sonar/gerrit/ReviewUtils.java

```diff
@@ -119,36 +129,28 @@ public static int maxLevel(ReviewInput reviewInput) {
         return lvl;
     }
 
+    private static String prefixKey(String prefix, String key) {
+        return prefix + "." + key;
+    }
+
+    public static String issueMessage(String originalMessage, Settings settings, Issue issue) {
+        HashMap<String, Object> valueMap = new HashMap<>();
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_IS_NEW), issue.isNew());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_RULE_KEY), issue.ruleKey());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_SEVERITY), issue.severity());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_MESSAGE), issue.message());
+        return substituteProperties(originalMessage, settings, valueMap);
+    }
+
 	public static String substituteProperties(String originalMessage, Settings settings) {
-	    String substitutedString = originalMessage;
```

Comment:
I noticed the substitution had been re-written and was not sure why as the libs we were using were already providing it

I'll put it back if you want me to, just wanted to "lighten the load" :)